### PR TITLE
GitHub actions

### PR DESCRIPTION
**Pull request by marcussc**
## Description
This PR introduces GitHub Actions.
CircleCI will not be disabled as of now, GitHub Actions will run in parallel to see how it performance and what needs to be tweaked for our needs.
Also:
- Update Fastlane
- Fix Linterwarnings for the latest SwiftLint 
- Cleanup/reordering of three Testplans (which will be the important ones in the future)
- Tweaks/Cleanup in Fastlane

### CODE REVIEW

File: src/xcode/fastlane/Fastfile [hunk 1]
```diff
@@ -70,6 +71,7 @@ platform :ios do
         disable_package_automatic_updates: true,
         result_bundle: true,
         code_coverage: true,
+        concurrent_workers: 7, # 8 cores make 7 concurrent tests right? :D
         number_of_retries: 1, #take this you flaky tests
```
Comment (ArturFriesen) [inline src/xcode/fastlane/Fastfile:75]: 😂

File: src/xcode/fastlane/Snapfile [hunk 2]
```diff
@@ -1,56 +0,0 @@
-# Uncomment the lines below you want to change by removing the # in the beginning
```
Comment (ArturFriesen) [inline src/xcode/fastlane/Snapfile:1]: Is this file not needed anymore?

File: src/xcode/ENA/TestPlans/UITests.xctestplan [hunk 3]
```diff
@@ -89,7 +89,51 @@
       "parallelizable" : true,
       "skippedTests" : [
         "ENAUITests",
-        "ENAUITests_11_QuickActions\/testLaunchViaShortcutFromFreshInstall()"
+        "ENAUITests_00_Onboarding\/test_0002_Screenshots_OnboardingFlow_EnablePermissions_normal_S()",
```
[the remark this answers is not in the document]
Comment (marcussc) [reply src/xcode/ENA/TestPlans/UITests.xctestplan:92]: Yes they are contained in the screenshots testplan and not needed in the UITest Testplan. The Screenshot and UI Test are run in parallel.

File: src/xcode/fastlane/Snapfile (hunk 2 above)
Comment (marcussc) [reply src/xcode/fastlane/Snapfile:1]: Nope its all inside the normal Fastfile. We had a split here before, some options set in the Snapfile and some in the Fastfile. I cleaned this up by moving it all to the Fastfile.

**Review by ArturFriesen — APPROVED** [review]

File: src/xcode/ENA/ENA/Source/Scenes/Home/Cells/Risk/HomeRiskTableViewCell.swift [hunk 4]
```diff
@@ -39,7 +39,6 @@ final class HomeRiskTableViewCell: UITableViewCell {
 
 			return accessibilityElements
 		}
-		// swiftlint:disable:next unused_setter_value
```
Comment (pascal-brause) [inline src/xcode/ENA/ENA/Source/Scenes/Home/Cells/Risk/HomeRiskTableViewCell.swift:42]: ❤️

File: src/xcode/ENA/TestPlans/UITests.xctestplan [hunk 5]
```diff
@@ -89,7 +89,51 @@
...
+        "ENAUITests_04a_ExposureSubmission\/test_screenshot_test_result_pending()",
+        "ENAUITests_04a_ExposureSubmission\/test_screenshot_test_result_positive()",
+        "ENAUITests_07_ContactJournal\/test_screenshot_AddTwoPersonsTwoLocations()",
+        "ENAUITests_07_ContactJournal\/test_screenshot_ContactJournalInformation()",
+        "ENAUITests_07_ContactJournal\/test_screenshot_EditLocationScreen()",
+        "ENAUITests_07_ContactJournal\/test_screenshot_EditPersonScreen()",
+        "ENAUITests_07_ContactJournal\/test_screenshot_TwoPersonsOneLocationAndMessages()",
+        "ENAUITests_09_TraceLocations\/test_screenshots_of_traceLocation_print_flow()",
+        "ENAUITests_10_CheckIns\/test_screenshot_WHEN_scan_QRCode_THEN_checkin_and_checkout()",
+        "ENAUITests_11_QuickActions\/testLaunchViaShortcutFromFreshInstall()",
```
Comment (pascal-brause) [inline src/xcode/ENA/TestPlans/UITests.xctestplan:119]: this one is a screenshot-test, too? the name gives no idea about it. Perhaps rename it?

File: src/xcode/ENA/TestPlans/UITests.xctestplan [hunk 6]
```diff
@@ -89,7 +89,51 @@
...
+        "ENAUITests_13_CreateHealthCertificate\/test_screenshot_RecoveryCertificateExpired()",
+        "ENAUITests_13_CreateHealthCertificate\/test_screenshot_TestCertificate()",
+        "ENAUITests_13_CreateHealthCertificate\/test_screenshot_shownConsentScreenAndDisclaimer()",
+        "ENAUITests_14_ValidateHealthCertificate\/test_screenshot_validation_country_picker()",
+        "ENAUITests_14_ValidateHealthCertificate\/test_screenshot_validation_datetime_picker()",
+        "ENAUITests_15_OnBehalfCheckinSubmission\/test_screenshot_OnBehalfCheckinSubmissionWithExistingTraceLocation()",
+        "ENAUITests_16_UniversalQRCodeScanner\/test_screenshot_UniversalQRCodeScannerAndQRScannerInfoScreen()",
+        "ENAUITests_19_TicketValidation\/test_screenshot_ticketValidation_failed()",
+        "ENAUITests_19_TicketValidation\/test_screenshot_ticketValidation_passed()",
+        "ENAUITests_20_CertificateReissuance"
```
Comment (pascal-brause) [inline src/xcode/ENA/TestPlans/UITests.xctestplan:136]: Only screenshot-tests in there, right?

File: src/xcode/ENA/TestPlans/AllTests.xctestplan [hunk 7]
```diff
@@ -89,83 +89,42 @@
     {
```
Comment (pascal-brause) [inline src/xcode/ENA/TestPlans/AllTests.xctestplan:89]: why are some of the screenshot tests removed but not everyone?
And what about the tests not including "screenshot" in the name, like e.g. "ENAUITests_14_ValidateHealthCertificate"?

File: src/xcode/ENA/TestPlans/UITests.xctestplan (hunk 5 above)
Comment (marcussc) [reply src/xcode/ENA/TestPlans/UITests.xctestplan:119]: Nope its not a Screenshot but take a look into the test:
`This test is INTENTIONALLY disabled`

**Review by pascal-brause — APPROVED** [review]

**Pull request merged**